simple_node:
  ros__parameters:
    topic_name_ : "simple_node"
    time_interval_: 1000
qos_node:
  ros__parameters:
    topic_name : "simple_node"
    timer_interval: 1000
